Packs/XSOARContentUpdateNotifications/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "XSOAR Content Update Notifications",
+    "description": "This pack will check for any available content updates for existing packs and send an e-mail or Slack message to users to inform them of the updates.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Adam Burt",
+    "url": "",
+    "email": "",
+    "created": "2021-06-15T14:34:33Z",
+    "categories": [],
```

Comment:
Choose a category please,  open an issue if you want a new category that is not listed

Reply:
Done